components/blog-article.tpl

```diff
@@ -12,7 +12,15 @@
   </header>
 
   <div class="article__content" data-search-indexing-allowed="true">
-    <div class="content-area content-area--article-excerpt content-area--overflowed-images"{% unless list_item %} {{ edy_intro_edit_text }}{% endunless %}>{% if list_item %}{{ article.excerpt }}{% else %}{% editable article.excerpt %}{% endif %}</div>
+    <div class="content-area content-area--article-excerpt content-area--overflowed-images"{% unless list_item %} {{ edy_intro_edit_text }}{% endunless %}>
+      {% if list_item %}{{ article.excerpt }}{% else %}{% editable article.excerpt %}{% endif %}
+    </div>
+
+    <div class="content-area">
```

Comment:
Wrapping `.readmore-link` to `.content-area` is semantically wrong. It is not a editable content area which user can edit with the rich text editor. The `.content-area` wrapper should only be a parent for `{% content %}`, `{{ article.excerpt }}`, `{{ article.body }}` etc.